content/blog/01_mapmaking.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "Map Making"
+date = 2021-05-01
+
+extra.image = "assets/images/blog/mapmaking.png"
++++
+
+Map Making 101
```

Comment:
This should probably be a title- also, an alternative title suggestion: "How We Make Our Maps"?